ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -134,13 +134,138 @@
       line: "Rhui Client package is not installed in the image"
       create: yes
       state: present
-    when: rhui_package_count == "0"
+    when: rhui_package_count == "0" and offer_type !='byol'
 
   - name: Set the variable if rhui client package is found
     set_fact:
       is_rhui_package_present : true
     when: rhui_package_count != "0"
 
+##Repo Validation Task
+
+- name: Check if offertype is equal to byol and client package is present
+  lineinfile:
+    path: "{{err_folder}}/err_msgs.log"
+    line: "\n  Client Package is present in byol Offertype Image"
+    create: yes
+    state: present 
+  when: offer_type =='byol' and rhui_package_count != "0"
+
+- name: Check the value of client package
```

Comment:
check this only if the above task is passed. also write it in the file else it will be missed in the pipeline

Reply:
's already visible in the pipeline, then not required